cli/azd/pkg/project/service_target_dotnet_containerapp.go

```diff
@@ -393,6 +396,26 @@ func (at *dotnetContainerAppTarget) Deploy(
 		if err != nil {
 			return nil, fmt.Errorf("deploying bicep template: %w", err)
 		}
+		// set the name of the container app to the name of the deployment
+		if len(deploymentResult.Resources) == 0 {
```

Comment:
I think this is probably okay for now. One thing to note is that this doesn't handle nested Bicep modules / ARM deployments IIRC.

Reply:
Another alternative if we want to make it similar to what `azd` is doing elsewhere today:

Honor the conventional `AZURE_CONTAINER_APP_ID` output variable -- this would solve it recursively, but will require user manual input.

Reply:
The definition for the ACA that Aspire provides is unlikely to bring more than one resource.  It currently uses a resource-group deployment and just defines the ACA creation.

I wanted to add just a little heads-up here in case this ever changes and/or there's a bug in Aspire.

Now, folks can still run azd infra synth and take control over the bicep that creates the ACA to bring more service's infra here, but that's a super edge case right now. So we don't care about this for now. :)